projects/Mallard/src/components/article/html/article.ts

```diff
@@ -58,10 +55,7 @@ const renderArticleContent = (
                 case 'media-atom':
                     return showMedia ? renderMediaAtom(el) : ''
                 case 'image': {
-                    const path = useImagePath({
-                        path: el.src.path,
-                        source: el.src.source,
-                    })
+                    const path = useImagePath(el.src)
```

Comment:
Keep a stable reference to an image to ensure we don't re-render on every call of this hook.